docs/building_blocks_for_flexible_content.md

```diff
@@ -0,0 +1,571 @@
+        molestie, dictum esta, mattis tellus.</p>
+```
+
+Theme: two thirds / one third
+
+```yaml
+- type: two_column_layout
+  theme: two_thirds_one_third
+  blocks:
+  - type: blocks_container
+    blocks:
+    - type: govspeak
+      content: |
+        <p>Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus.</p>
+    - type: heading
+      content: "Title of content"
+    - type: image
+      theme: full_width
+      src: "landing_page/placeholder/960x640.png"
+      alt: "Placeholder image"
+    - type: govspeak
+      content: |
+        <p>Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus.</p>
+```
+
+The left column is always populated if only one column is defined.
+A blocks container applied to a column so that everying in the column can be styled relative to one and other.
+
+## Indexing block content in search
+
+Only content in blocks that has a label of `content` will be indexed in site search.
+
+```yaml
+blocks:
+- type: govspeak
+  content: |
+    ## This will be a searchable header
+    here's some content that's searchable
+- type: button
+  text: Press me (not searchable)
+- type: header
+  content: searchable header
+```
+
+In the example above only the text in the govspeak and header blocks will be searchable.
+
+## Navigation Groups
+
+The top-level key `navigation_groups` contains an array of items with the keys `id` and `links`. The id is used to identify the group for navigation blocks like [main navigation](#main-navigation) and [side navigation](#side-navigation), which reference a navigation group with their `navigation_group_id` key. The links key is an array of `text`/`href` values, although a links item can itself contain a `links` key pointing to an array of links for a submenu (main navigation supports sub menus, side navigation does not)
+
+```yaml
+navigation_groups:
+- id: Top Menu
+  links:
+  - text: Goals
+    href: /landing-page/goals
+  - text: Tasks
+    href: /landing-page/tasks
+    links:
+      - text: Be kinder
+        href: "/landing-page/be-kinder"
+      - text: Exercise more
+        href: "/landing-page/exercise-more"
+      - text: Donate to charity
+        href: "/landing-page/donate-to-charity"
+      - text: Learn something new
+        href: "/landing-page/learn-something-new"
+      - text: Be thankful
+        href: "/landing-page/be-thankful"
+```
+
+## Extending a page (Whitehall Only)
+
+When editing a page in Whitehall, you may wish to use data from another page. Adding the top-level key `extends`, and the path of another landing page, will cause the data from the current block to be merged over the data from the extended block before publishing. This is a shallow merge (ie only top-level values are checked), and is currently used so that navigation groups can be specified in one page and used by sub-pages.
```

Comment:
Suggestion: For clarity change "you may wish to use data from another page" to "you may wish to use data from another flexible content page" as there are many things on GOV.UK called pages.